### PR TITLE
Trigger pageview hitType for GA

### PR DESCRIPTION
**Pull request by craigmulligan**
Connects to #24 
docs: https://developers.google.com/analytics/devguides/collection/analyticsjs/single-page-applications

Change-Type: patch

### CODE REVIEW

**Review by thgreasi — APPROVED** [review]
LGTM but you should probably wait for a second approval

**Review by pimterry — APPROVED** [review]

**Pull request merged**
